app/views/artists/show.html.erb

```diff
@@ -1,18 +1,66 @@
+<div class="show-page-wrapper">
+
 <div class="container">
   <h2>Showing artist details</h1>
 
-  <div class="card" style="width: 18rem;">
-  <img src=<%= @artist.image_url %> class="card-img-top" alt="artist picture">
-  <div class="card-body">
-    <h5 class="card-title"><%= @artist.name %></h5>
-    <p class="card-text"><%= @artist.description %></p>
+  <div class="-">
```

Comment:
what's this? that's a very odd name for a class name.

Reply:
I think I just wanted to remove the class, I will double-check it

Reply:
I removed the dash!